solc/CommandLineInterface.cpp

```diff
@@ -670,6 +682,11 @@ Allowed options)",
 			"<libraryName>:<address> [, or whitespace] ...\n"
 			"Address is interpreted as a hex string optionally prefixed by 0x."
 		)
+		(
+				g_strRevertStrings.c_str(),
+				po::value<string>()->value_name(boost::join(g_revertStringsArgs, ",")),
+				"Strip revert reason strings or add additional debugging information."
```

Comment:
Might we want to list and explain the individual values?

Reply:
This is just the help screen, I think people should use the documentation for that.